backend/app/graphql/onboarding_request.py

```diff
@@ -31,7 +28,103 @@ class OnboardingRequest(graphene.ObjectType):
     status = graphene.String()
 
 
+# Queries
+
+# Return object for queries
+class GetOnboardingRequest(Query):
+    contact_name = graphene.String()
+    contact_email = graphene.String()
+    contact_phone = graphene.String()
+    role = graphene.String()
+    date_submitted = graphene.DateTime()
+    status = graphene.String()
+
+
+class OnboardingRequestQueries(QueryList):
+    getAllOnboardingRequests = graphene.List(
+        GetOnboardingRequest,
+        first=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+        role=graphene.String(default_value=""),
+        status=graphene.String(default_value=""),
+    )
+    getOnboardingRequestById = graphene.List(
+        GetOnboardingRequest,
+        id=graphene.String(),
+        first=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+    )
+
+    def resolve_getAllOnboardingRequests(self, info, first, offset, role, status):
+        requests = services["onboarding_request_service"].get_all_onboarding_requests()
+        if role != "":
+            filtered = []
+            for request in requests:
+                if request["info"]["role"] == role:
+                    filtered.append(
+                        GetOnboardingRequest(
+                            contact_name=request["info"]["contact_name"],
+                            contact_email=request["info"]["contact_email"],
+                            contact_phone=request["info"]["contact_phone"],
+                            role=request["info"]["role"],
+                            date_submitted=request["date_submitted"],
+                            status=request["status"],
+                        )
+                    )
+            return filtered[offset : offset + first]
+
+        if status != "":
+            filtered = []
+            for request in requests:
+                if request["status"] == status:
+                    filtered.append(
+                        GetOnboardingRequest(
+                            contact_name=request["info"]["contact_name"],
+                            contact_email=request["info"]["contact_email"],
+                            contact_phone=request["info"]["contact_phone"],
+                            role=request["info"]["role"],
+                            date_submitted=request["date_submitted"],
+                            status=request["status"],
+                        )
+                    )
+            return filtered[offset : offset + first]
+
+        return [
+            *map(
```

Comment:
It would also be best to have the service convert the results to DTO format ahead of time, so the transformation is simpler here.